test/router-options.test.js

```diff
@@ -131,12 +88,20 @@ test('Should honor maxParamLength option', t => {
 })
 
 test('Should expose router options via getters on request and reply', t => {
-  t.plan(7)
+  t.plan(8)
   const fastify = Fastify()
+  const expectedSchema = {
+    params: {
+      id: { type: 'integer' }
+    }
+  }
 
-  fastify.get('/test/:id', (req, reply) => {
-    t.equal(reply.context.config.url, '/test/:id')
-    t.equal(reply.context.config.method, 'GET')
```

Comment:
I would keep these two lines: they will fail when we will remove the `context` option but till then we need to test it

Reply:
Sure, done in [f3be7b5](https://github.com/fastify/fastify/pull/4216/commits/f3be7b547b7708e104136c93ca3140f32eabd241)